<?xml version="1.0" encoding="UTF-8"?><testrun name="TestMergeSortIter2" project="BT-Library-Java" tests="5" started="5" failures="0" errors="4" ignored="0">
  <testsuite name="bt.tests.core.algorithms.sorting.TestMergeSortIter2" time="0.009">
    <testcase name="test_MergeSortIter2_012345678_Expect_012345678_True" classname="bt.tests.core.algorithms.sorting.TestMergeSortIter2" time="0.002">
      <error>bt.core.exceptions.EmptyQueueException&#13;
	at bt.core.collections.dataTypes.Queue.dequeue(Queue.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.mergeSort(MergeSortIter2.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.sort(MergeSortIter2.java:19)&#13;
	at bt.tests.core.algorithms.sorting.TestMergeSortIter2.test_MergeSortIter2_012345678_Expect_012345678_True(TestMergeSortIter2.java:153)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_MergeSortIter2_63124805_10_7_9_15_11_13_12_14_Expect_0123456789101112131415_True" classname="bt.tests.core.algorithms.sorting.TestMergeSortIter2" time="0.001">
      <error>bt.core.exceptions.EmptyQueueException&#13;
	at bt.core.collections.dataTypes.Queue.dequeue(Queue.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.mergeSort(MergeSortIter2.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.sort(MergeSortIter2.java:19)&#13;
	at bt.tests.core.algorithms.sorting.TestMergeSortIter2.test_MergeSortIter2_63124805_10_7_9_15_11_13_12_14_Expect_0123456789101112131415_True(TestMergeSortIter2.java:100)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_MergeSortIter2_63124805_Expect_01234568_True" classname="bt.tests.core.algorithms.sorting.TestMergeSortIter2" time="0.001">
      <error>bt.core.exceptions.EmptyQueueException&#13;
	at bt.core.collections.dataTypes.Queue.dequeue(Queue.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.mergeSort(MergeSortIter2.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.sort(MergeSortIter2.java:19)&#13;
	at bt.tests.core.algorithms.sorting.TestMergeSortIter2.test_MergeSortIter2_63124805_Expect_01234568_True(TestMergeSortIter2.java:46)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_MergeSortIter2_876543210_Expect_012345678_True" classname="bt.tests.core.algorithms.sorting.TestMergeSortIter2" time="0.001">
      <error>bt.core.exceptions.EmptyQueueException&#13;
	at bt.core.collections.dataTypes.Queue.dequeue(Queue.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.mergeSort(MergeSortIter2.java:25)&#13;
	at bt.core.algorithms.sorting.MergeSortIter2.sort(MergeSortIter2.java:19)&#13;
	at bt.tests.core.algorithms.sorting.TestMergeSortIter2.test_MergeSortIter2_876543210_Expect_012345678_True(TestMergeSortIter2.java:188)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="test_MergeSortIter2_empty_Expect_empty_catchExc_false" classname="bt.tests.core.algorithms.sorting.TestMergeSortIter2" time="0.003"/>
  </testsuite>
</testrun>
